addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js

```diff
@@ -99,6 +110,54 @@ export class SpreadsheetDashboardAction extends Component {
         return dashboard.model.getters.getGlobalFilters();
     }
 
+    /**
+     * @returns {object[]}
+     */
+    get cogMenuItems() {
+        return [
+            {
+                id: "edit",
+                label: _t("Edit"),
+                icon: "fa fa-pencil",
+                action: (id) => this.editDashboard(id),
+                isVisible: true,
+            },
+            {
+                id: "delete",
+                label: _t("Delete"),
+                icon: "fa fa-trash",
+                action: (id) => this.deleteDashboard(id),
+                isVisible: this.isDashboardAdmin,
```

Comment:
The whols cog menu should be invisible if the user is not dashboard admin right ? A non-admin cannot edit the dashboards either

Reply:
No, currently only 'Delete' option is hidden for non-admin users, similar to how the 'Edit' button is still visible in the configuration for non-admins but opens in read-only mode. We can reach out to @Francois-frgi to clarify this behavior.